src/Symfony/Bundle/FrameworkBundle/Resources/config/session.php

```diff
@@ -91,12 +92,17 @@
             ->factory([SessionHandlerFactory::class, 'createHandler'])
             ->args([abstract_arg('A string or a connection object')])
 
+        ->set('session.collector', Closure::class)
```

Comment:
this should this be moved to the `collectors.php` file, isn't it?
what about naming this like `data_collector.request.session_collector`?